### ISSUE TITLE
[false positive] json_decode inspection does not recognize name identifier for the flags argument

### ISSUE THREAD

**Issue by winkbrace** [kalessil/phpinspectionsea#1807]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) |
| Language level |  PHP 8.0                  |

Current behaviour
--

This line uses name identifiers to skip the default 2nd and 3rd argument.
Unfortunately the inspection seems to look only for a 4th argument, as it is advising to harden the errors handling, which is already there.

```php
$content = json_decode(json: $client->getResponse()->getContent(), flags: JSON_THROW_ON_ERROR);
```

![harden errors handling](https://user-images.githubusercontent.com/1883930/152803718-8176edfd-5540-43e9-b840-ebaaf3872db1.png)

Expected behaviour
--

It should recognize the name identifier `flags` contains the desired `JSON_THROW_ON_ERROR`.

Environment details
---

PHP 8.0
PhpStorm 2021.3.1
Php Inspections (EA Extended) 4.0.7.1

**Comment by jdreesen** [comment]
Duplicate of #1725

**Comment by winkbrace** [comment]
Ah, thanks. I didn't find it when looking for `name identifier` :)
